wizeline/remix-project-lab
# Feat/210/GitHub weekly participation by project

# What does this PR do?
Get the weekly participation by project statistics of github.


# How should this be manually tested?

1. make sure at least one project has its repository url registered
2. Run the _get-gitprojects-activity_ script
3. Go to Git hub info page of the project
4. Check the statistics /github-info



# Any background context you want to provide?



# What are the relevant tickets?
#210 


# Screenshots
![image](https://github.com/wizeline/remix-project-lab/assets/74632242/cdecf5af-e423-4218-90fb-0e95c8abe195)

![image](https://github.com/wizeline/remix-project-lab/assets/74632242/548995e3-0803-4201-837f-93ce0b47b584)




# Questions



# Checklist
